test/test-downloader.rb

```diff
@@ -0,0 +1,29 @@
+class DownloaderTest < Test::Unit::TestCase
```

Comment:
Nothing below this line has been changed in this file.

Reply:
We can remove this file.